.github/workflows/create_dashboard.yaml

```diff
@@ -0,0 +1,84 @@
+name: Create Dashboard
```

Comment:
Do dashboards also still need to be created (as with the apply of the `.identity` in PROD)?

Reply:
It seems that it is still required since that pipeline updates the dashboard every time the swagger is updated. The `.identity` content is related only for GitHub Action authorizations.